src/plugins/resolver/CMakeLists.txt

```diff
@@ -3,22 +3,41 @@ include(LibAddPlugin)
 set (SOURCES resolver.h resolver.c filename.c)
 
 find_package(Threads)
+find_package(LibRt)
+
 
 #TODO: remove "default" resolver
-add_plugin(resolver
-	SOURCES
-		${SOURCES}
-	LINK_LIBRARIES
-		${CMAKE_THREAD_LIBS_INIT}
-		${CMAKE_REALTIME_LIBS_INIT}
-	COMPILE_DEFINITIONS
-		ELEKTRA_VARIANT_BASE=\"\"
-		ELEKTRA_VARIANT_USER=\"hpu\"
-		ELEKTRA_VARIANT_SYSTEM=\"b\"
-		ELEKTRA_PLUGIN_NAME=\"resolver\"
-		ELEKTRA_LOCK_FILE
-		ELEKTRA_LOCK_MUTEX
+if (LIBRT_FOUND)
+	add_plugin(resolver
+		SOURCES
+			${SOURCES}
+		LINK_LIBRARIES
+			${CMAKE_THREAD_LIBS_INIT}
+			${LIBRT_LIBRARIES}
```

Comment:
This could be written which much less duplication by the using an `if` outside the `add_plugin`.